web/app.js

```diff
@@ -730,6 +732,21 @@ const PDFViewerApplication = {
     return this.externalServices.supportedMouseWheelZoomModifierKeys;
   },
 
+  get useUserDefinedColors() {
```

Comment:
Maybe `allowDocumentColorUse` instead here, to have slightly more consistent method names between the the `app.js`, `firefoxcom.js`, and `PdfStreamConverter.jsm` files?